torch/_library/custom_ops.py

```diff
@@ -187,6 +192,55 @@ def _qualname(self) -> str:
     def __repr__(self) -> str:
         return f"<CustomOpDef({self._qualname})>"
 
+    @contextmanager
+    def set_kernel_enabled(self, device_type: str, enabled: bool = True):
```

Comment:
I don't think we have docs for CustomOpDef on the website. Could you figure out how to add them sometime? (doesn't have to be in this PR).

In the past the APIs we have on CustomOpDef are the same as the ones in torch.library.*, but now we have a CustomOpDef specific API.